pkg/keys/constants.go

```diff
@@ -92,6 +92,8 @@ var (
 	// LocalRaftTruncatedStateSuffix for the corresponding unreplicated
 	// RaftTruncatedState.
 	_ = []byte("rftt")
+	// LocalRangeGCHintSuffix is the suffix for the GC hint timestamp.
```

Comment:
nit: the GC hint isn't necessarily just a timestamp, since the struct can be extended.